bookwyrm/models/book.py

```diff
@@ -82,10 +86,14 @@ class Book(BookDataModel):
     cover = fields.ImageField(
         upload_to="covers/", blank=True, null=True, alt_field="alt_text"
     )
+    preview_image = models.ImageField(
+        upload_to="previews/covers/", blank=True, null=True
+    )
     first_published_date = fields.DateTimeField(blank=True, null=True)
     published_date = fields.DateTimeField(blank=True, null=True)
 
     objects = InheritanceManager()
+    field_tracker = FieldTracker(fields=["authors", "title", "subtitle", "cover"])
```

Comment:
This will help us track which field are modified. In the Book model, the image generation process shouldn't be called for any field change other than `authors`, `title`, `subtitle`, `cover`.